### ISSUE TITLE
Undefined index: leads_id in AOR_Report.php on line 829

### ISSUE THREAD

**Issue by connorshea** [salesagility/SuiteCRM#7868]
#### Issue
There's a notice about an undefined index in some code in `AOR_Report.php`

#### Expected Behavior
No notices in PHP Error Log

#### Actual Behavior

```
[13-Sep-2019 23:26:41 UTC] PHP Notice:  Undefined index: leads_id in /Users/connorshea/code/sites/SuiteCRM/modules/AOR_Reports/AOR_Report.php on line 829
```

This is the line that errors:
https://github.com/salesagility/SuiteCRM/blob/1d2acdf7b0c7375557b93f38475b2881343f920f/modules/AOR_Reports/AOR_Report.php#L829

#### Possible Fix
We should check to make sure the `$row[$att['alias'] . '_id']` value exists before using it.

#### Steps to Reproduce
I'm not 100% sure how to reproduce it, but I think you can do it by running reports.
1. Create a report involving the Leads module
2. Run the report (you may need to download it as a PDF?)
3. Check the logs and see if there's an error like `Undefined index: leads_id`

#### Context
It's just adding annoying notices to our logs.

#### Your Environment
* SuiteCRM Version used: 7.10.19
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Firefox 69
* Environment name and version (e.g. MySQL, PHP 7): MySQL 5.7, PHP 7.2
* Operating System and version (e.g Ubuntu 16.04): macOS Mojave 10.14.6

**Comment by connorshea** [comment]
I think this was reported in #7604, and was supposed to be fixed by #6794, but it doesn't look like it was fully fixed, seeing as I have the new code from #6794 and it still happens :)

**Comment by connorshea** [comment]
I can fix it by changing this:

```php
$params = array('record_id' => $row[$att['alias'] . '_id']);
```

to this:

```php
if (isset($row[$att['alias'] . '_id'])) {
    $params = array('record_id' => $row[$att['alias'] . '_id']);
} else {
    $params = [];
}
```

It might be more optimal to make `$att['alias'] . '_id'` a variable so it doesn't need to be created twice, but that's a micro-optimization so I don't care too much.

I'm not sure if this causes any problems since the `$params` variable is used as a parameter in `getModuleField` right afterward, but it doesn't seem to change anything about the reports that are rendered.
